test_scripts/AppServices/RPCPassThrough/005_PTU_allow_unknown_rpc_passthrough.lua

```diff
@@ -0,0 +1,48 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Request to renumber this test to `008` since I am using `005_SendLocation_missing_permissions`, `006_ButtonPress_Handled_request`,`007_ButtonPress_missing_permissions`